src/executables/util_cs.c

```diff
@@ -2300,23 +2332,34 @@ paramdump (UTIL_FUNCTION_ARG * arg)
       goto error_exit;
     }
 
-  if (both_flag)
+#if defined(SA_MODE)
+  if (envvar_get ("FOR_QA"))
+    {
+      fprintf (outfp,
+	       msgcat_message (MSGCAT_CATALOG_UTILS, MSGCAT_UTIL_SET_PARAMDUMP, PARAMDUMP_MSG_STANDALONE_PARAMETER));
+      sysprm_dump_parameters (outfp, ' ', PRM_FOR_CLIENT | PRM_FOR_SERVER, PRM_OR_CONDITION, out_flags,
+			      PRM_OR_CONDITION);
+    }
+  else
     {
+#endif
```

Comment:
```suggestion
#endif
        {
```
이렇게 하면 아래 2358 라인의 #if defined(SA_MODE)도 필요 없습니다.

Reply:
리뷰감사합니다. 반영하겠습니다.